openmc/settings.py

```diff
@@ -566,6 +588,35 @@ def statepoint(self, statepoint):
                                  "statepoint options.".format(key))
         self._statepoint = statepoint
 
+    @surf_src_read.setter
+    def surf_src_read(self, surf_src_read):
+        cv.check_type('surface source writing options', surf_src_read, Mapping)
```

Comment:
Is this meant to be `'surface source reading options'`?

```suggestion
        cv.check_type('surface source reading options', surf_src_read, Mapping)
```